core/src/main/scala/org/apache/spark/sql/CachedDataFrame.scala

```diff
@@ -272,18 +309,24 @@ class CachedDataFrame(df: Dataset[Row], val queryString: String,
       newpls: mutable.ArrayBuffer[ParamLiteral]): Unit = {
     if (allbcplans.nonEmpty && !firstAccess) {
       allbcplans.foreach { case (bchj, refs) =>
+        println(s"Repreparing for bcplan = ${bchj} with new pls = ${newpls.toSet}")
```

Comment:
Still comment out the println because it breaks scalastyle. Too many reds in IDEA now in our code. Will be turning on scalaStyle by default for core, cluster, aqp soon.